consensus/snapshot/BUILD

```diff
@@ -1,14 +1,35 @@
+genrule(
+    name = "snapshot_files",
+    outs = [
+        "snapshotMainnet.txt",
+        "snapshotMainnet.sig",
+        "snapshotTestnet.txt",
+        "snapshotTestnet.sig",
+    ],
+    cmd =
+        """
+        url="https://raw.githubusercontent.com/iotaledger/iri/master/src/main/resources"
```

Comment:
Could you replace the wgets etc with https://docs.bazel.build/versions/master/be/workspace.html#http_file ?

Reply:
This was very temporary, do we want to get this PR merged before the snapshot files repository exists ?